### PR TITLE
[Mac manager] Support Ignite repository and update DB schema

### PR DESCRIPTION
**Pull request by kimeunju108**
The major changes in this PR:

* Repository was changed to Ignite from Redis.

* Repository schema update: mac_state, mac_range, mac_pool

* Performance improvement: Mac addresses are generated in advance and wait to be allocated to ports in mac pool.

### CODE REVIEW

File: services/mac_manager/pom.xml [hunk 1]
```diff
@@ -88,11 +88,7 @@
             <groupId>io.springfox</groupId>
             <artifactId>springfox-swagger2</artifactId>
             <version>2.6.1</version>
-        </dependency>
-        <dependency>
-            <groupId>io.springfox</groupId>
-            <artifactId>springfox-swagger-ui</artifactId>
-            <version>2.2.2</version>
```
Comment (xieus) [inline services/mac_manager/pom.xml:95]: Let us keep io.springfox as they are used to generate Swagger doc/UI

File: services/mac_manager/src/test/java/com/futurewei/alcor/macmanager/swagger/SwaggerConfig.java [hunk 2]
```diff
@@ -14,7 +14,7 @@
         limitations under the License.
 */
 
-package com.futurewei.alcor.macmanager.config;
+package com.futurewei.alcor.macmanager.swagger;
```
Comment (xieus) [inline services/mac_manager/src/test/java/com/futurewei/alcor/macmanager/swagger/SwaggerConfig.java:17]: No need to touch swagger file as they are commonly used by other services as well for Swagger UI generation.

Comment (kimeunju108) [reply services/mac_manager/src/test/java/com/futurewei/alcor/macmanager/swagger/SwaggerConfig.java:17]: Yes, I will not touch the file.

File: services/mac_manager/pom.xml (hunk 1 above)
Comment (kimeunju108) [reply services/mac_manager/pom.xml:95]: Yes, I will keep io.springfox

File: services/mac_manager/src/test/java/com/futurewei/alcor/macmanager/swagger/SwaggerConfig.java (hunk 2 above)
Comment (xieus) [reply services/mac_manager/src/test/java/com/futurewei/alcor/macmanager/swagger/SwaggerConfig.java:17]: Please revert this change.

**Comment by xieus** [comment]
@kimeunju108 Since the mac allocation algorithm and schema is changed in this PR, please remember to update the design doc (should modified in a separate PR).

**Pull request merged**